### ISSUE TITLE
Question regarding to the newly added NMS step before selecting samples

### ISSUE THREAD

**Issue by sjiang17** [weiliu89/caffe#447]
### Issue summary
Hi, I noticed that you have added an NMS step before selecting samples in the new version of code, but it was not enabled in your sample code. Have you tried using it and was the performance better?

Thanks.

**Comment by weiliu89** [comment]
Yes. It was trying to reproduce the OHNM work from Ross Girsick. However I didn't find it better than simply take the max negative samples, and it is slower since nms is slow. Maybe I didn't do it right, I am not sure. You could try and let me know your findings.

**Comment by sjiang17** [comment]
Thanks for the reply @weiliu89 . I was also wondering why did you use priors (instead of predicted bboxes) for matching? How much worse was the performance of using predicted bboxes?

**Comment by weiliu89** [comment]
The first version of the paper has a discussion about this. You can check out for more details.
